resources/common.js

```diff
@@ -104,8 +104,8 @@ export const Util = (() => {
 })();
 
 (function() {
-  // Exit early if running within Node
-  if (typeof location === 'undefined')
+  // Exit early if running within Node or within a web worker
+  if (typeof location === 'undefined' || (typeof WorkerLocation !== 'undefined' && location instanceof WorkerLocation))
```

Comment:
Hmm! I am not 100% sure offhand what the right way to do this.  I saw somebody else checking `WorkerGlobalScope ` as a way to do this: https://gist.github.com/miguelmota/a0c9aa9398507e606a6e

Also, I do wonder if `typeof location === 'undefined` should be `typeof window === 'undefined'` at the very least to match `EventBus.js` below as a consistent way to detect node vs browser.

Reply:
For the `EventBus.js` case, it may make more sense to change it to use [globalThis](https://caniuse.com/?search=globalThis) instead, as such:

```js
scope = scope !== undefined ? scope : globalThis;
```

It should also be fine to change the `common.js` check to:

```js
  // Exit early if not running within a `Window` scope
  if (typeof Window === 'undefined' || !globalThis instanceof Window)
```

Thoughts?

Reply:
Seems reasonable to me! I don't think I am confident enough here to know what the "right" way to check this is.